### ISSUE TITLE
Can't Lock or configure omnibar

### ISSUE THREAD

**Issue by jbo616** [jordonwow/omnibar#78]
I cant configure omni bar1 or bar2 nor can i configure in anyway.

i have tried deleting lua and re installing the addon, but still nothing works.

are there any fixes for this?

**Comment by dbwtf** [comment]
Same

**Comment by yaerox** [comment]
Same problem for me since PrePatch went live. Reinstalled Addon twice. Also deleted complete Interface+WTF Folder.
![image](https://user-images.githubusercontent.com/3898071/98034585-820e7a00-1e17-11eb-9630-662e8c20d8ac.png)

**Comment by AndrzejWroGM** [comment]
+1

**Comment by rolandmeo** [comment]
+1

**Comment by hleyg** [comment]
Same, really annoying :D 👍

**Comment by dll94** [comment]
the addon is probably dead/abandoned

**Comment by mgdanglade** [comment]
Hi. 

I have solved it by installing an older version. Exactly 9.5. With this version you can add spells and later update the addon to the latest version.

**Comment by jbo616** [comment]
How do I get hold of older versions?


Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Comment by 36Crazyfists** [comment]
Using version 9.5-nolib worked for me. I am using the new CurseForge addon client to manage my WoW addons. Here is where in CurseForge to install that specific version:
![image](https://user-images.githubusercontent.com/20711899/98870554-1345a800-2439-11eb-8b92-7fc744a14bf9.png)

Here is what the addon looks like in game now:
![image](https://user-images.githubusercontent.com/20711899/98870646-307a7680-2439-11eb-8474-4e9f5c9513b8.png)

I did not try other versions, so I am not sure if newer ones work. I just saw @mgdanglade's comment and tried this specific version first and it worked for me.
